packages/steppp/index.html

```diff
@@ -8,56 +8,85 @@
 
     <style>
       * {
-        color: white;
+        color: black;
       }
```

Comment:
a little spacey-boi here

```suggestion
        color: black;
      }

```